core/editor/editor.php

```diff
@@ -621,21 +621,21 @@ public function enqueue_scripts() {
 				'manage_global_colors' => __( 'Manage Global Colors', 'elementor' ),
 				'create_global_color' => __( 'Create New Global Color', 'elementor' ),
 				'delete_global_color' => __( 'Delete Global Color', 'elementor' ),
-				'delete_global_color_info' => __( 'Please note that by deleting a global color, all of its instances will inherit their value from an unknown source', 'elementor' ),
-				'global_colors_info' => __( 'Global colors help you work smarter. Save a color, and use it anywhere throughout your site. Access and edit your global colors by clicking the Manage button below.', 'elementor' ),
+				'delete_global_color_info' => __( 'You\'re about to delete a Global Color. Note that if it\'s being used anywhere on your site, it will inherit a default color.', 'elementor' ),
```

Comment:
@shilo-ey it needs also a sentence  about refresh the page?

Reply:
It's irrelevant to this PR

Reply:
Let's refresh the preview without prompting. It would prevent issues and will reduce actions needed to be made by our users.
@matipojo